src/components/player/player.tsx

```diff
@@ -432,6 +410,12 @@ export class Player {
   public async seek(seconds: number) {
     await this._invokePlayerFunction('seek', [seconds]);
 
+    this.status = {
+      ...this.status,
+      progress: { ...this.status.progress, seconds },
+    };
+    this.cueUpdate(seconds);
```

Comment:
So if `seek` triggers a `timeUpdate`, do we need to call `cueUpdate` here?

Reply:
`timeUpdate` is triggered when Kaltura or Vimeo emits a `timeupdate` event, which is usually when seeking except when the video has not yet been started. So these lines (413 to 417) are only necessary to update the cue or the status in a video in which play has not been pressed.

Reply:
So it would work if we make it dependent to `this.status.mode === Mode.PAUSED` to avoid unnecessary `cueUpdates`? Or do you mean it's only if a video has never been started at all?

Reply:
yes, that's a good idea. If I move it inside the `Mode.PAUSE` conditional it works when the video has never started and we will only be updating twice when the video is paused but it was already started at some point.